src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public static final Color WHITE = Color.WHITE;
+    public static final int DEFAULT_FIGURE_RADIUS = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int type = (int) (Math.random() * 5);
```

Comment:
use constants instead of numbers
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)